pip_review/__main__.py

```diff
@@ -29,12 +24,13 @@ def check_output(*args, **kwargs):
             raise error
         return output
 
-try:
+else: # Python2 Imports
+    import urllib2 as urllib_request
+    from subprocess import check_output
     import __builtin__
-    input = getattr(__builtin__, 'raw_input')  # Python2
-except (ImportError, AttributeError):
-    pass
+    input = getattr(__builtin__, 'raw_input')
 
+from pkg_resources import parse_version
 from packaging import version
```

Comment:
While you're at it, to complete the neatness, would you mind moving these two lines above the if/else block? Thanks in advance.